src/components/General/KBreadcrumb.vue

```diff
@@ -49,6 +49,6 @@ export default {
     padding 0 16px
     font-weight 500
     text-transform uppercase
-  .lastElement
+body.desktop .lastElement
```

Comment:
How does this work when it is scoped? the `body` element is not in this component?

Reply:
Works just the same, it's only that the element with the class "lastElement" has to be in the same template

Reply:
Ok, seems fine, just confused me a bit.